dev/io.openliberty.mcp.internal/test/io/openliberty/mcp/internal/testutils/MockAnnotatedMethod.java

```diff
@@ -58,8 +58,12 @@ public Method getJavaMember() {
 
     /** {@inheritDoc} */
     @Override
-    protected AnnotatedElement getAnnotatedElement() {
+    public AnnotatedElement getAnnotatedElement() {
         return method;
     }
 
+    public String getName() {
+        return method.getName();
+    }
```

Comment:
Unsure of the purpose of these changes.

Since this is a mock, these methods won't be available on the real object.

Reply:
Looks like we used to call `ToolMetadata.name()` in `SchemaTest` and were then searching for the tool name?

If we now want to search by method name, it makes more sense to filter on `Method.getName` before creating the `MockAnnotatedMethod`.